setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="cashctrl_ledger",
+    version="v0.0.1",
+    description="Python package that implements the pyledger.LedgerEngine and integrates with cashctrl",
+    url='https://github.com/macxred/cashctrl_ledger',
+    author="Lukas Elmiger",
```

Comment:
@AlexTheWizardL: Don't you want to be the package author?

Reply:
Yeah, sure. Let's have `author="Lukas Elmiger, Oleksandr Stepanenko"`